pkg/utils/common.go

```diff
@@ -59,7 +60,7 @@ func GetUkiCmdline() []string {
 	cmdlineOverride := viper.GetStringSlice("cmdline")
 	if len(cmdlineOverride) == 0 {
 		// For no extra cmdline, we default to install mode
-		return []string{constants.UkiCmdline + " " + constants.UkiCmdlineInstall}
+		return []string{constants.UkiCmdlineInstall + " " + constants.UkiCmdline}
```

Comment:
I think this might have been why we were seeing the weird names :P

Reply:
I don't recall doing this :D. Let me revert it.

Reply:
![image](https://github.com/kairos-io/enki/assets/2794419/4749bdaa-3b65-42dd-9d6c-0820dfbf3e7d)
I'm not sure if that made any change

Reply:
weeeeird